### ISSUE TITLE
URL of data sources

### ISSUE THREAD

**Issue by meehkal** [clics/clics1#23]
Interessantes Projekt! Bitte überprüft die Links auf der Webseite (http://clics.lingpy.org/languages.php), z.B. http://wold.livingsources.org/vocabulary/14 --> http://wold.clld.org/vocabulary/14. M. Rießler

**Comment by LinguList** [comment]
Ja, vielen Dank für den Hinweis! Wir arbeiten an einer neuen Auflage mit
der Integration aller neuen Daten (auch mehr Daten). Da wird dann
hoffentlich alles korrigiert.
